src/main/resources/env-map.yml

```diff
@@ -1,7 +1,14 @@
 env_vars:
-  CONNECTOR_URL: '."user-provided"[0].credentials.card_connector_url'
-  CONNECTOR_DD_URL: '."user-provided"[0].credentials.directdebit_connector_url'
-  PUBLIC_AUTH_URL: '."user-provided"[0].credentials.publicauth_api_path_url'
-  LEDGER_URL: '."user-provided"[0].credentials.ledger_url'
-  PUBLICAPI_BASE: '."user-provided"[0].credentials.publicapi_url'
+  CONNECTOR_URL: '.[][] | select(.name == "app-catalog")                                 | .credentials.card_connector_url'
+  CONNECTOR_DD_URL: '.[][] | select(.name == "app-catalog")                              | .credentials.directdebit_connector_url'
+  PUBLIC_AUTH_URL: '.[][] | select(.name == "app-catalog")                               | .credentials.publicauth_api_path_url'
+  LEDGER_URL: '.[][] | select(.name == "app-catalog")                                    | .credentials.ledger_url'
+  SENTRY_DSN: '.[][] | select(.name == "publicapi-secret-service")                       | .credentials.sentry_dsn'
+  RATE_LIMITER_VALUE: '.[][] | select(.name == "publicapi-secret-service")               | .rate_limiter_value'
```

Comment:
The vars for the secret service are also under `credentials` looking at publicapi with bound secret service...
```
    "credentials": {
     "rate_limiter_elevated_accounts": "31",
```